notebooks/concurrent_tasks.ipynb

```diff
@@ -0,0 +1,1224 @@
+{
```

Comment:
<p>I'm never sure of these sections, the hrefs may not work when we merge it. If it doesn't we'll fix later :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>